src/account/azext_account/generated/_params.py

```diff
@@ -6,43 +6,52 @@
-        c.argument('offer_type', arg_type=get_enum_type(['MS-AZR-0017P', 'MS-AZR-0148P']), help='The offer type of the subscription. For example, MS-AZR-0017P (EnterpriseAgreement) and MS-AZR-0148P (EnterpriseAgreement devTest) are available. Only valid when creating a subscription in a enrollment account scope.')
-
-    with self.argument_context('account subscription create-csp') as c:
-        c.argument('billing_account_name', help='The name of the Microsoft Customer Agreement billing account for which you want to create the subscription.')
-        c.argument('customer_name', help='The name of the customer.')
-        c.argument('display_name', help='The friendly name of the subscription.')
-        c.argument('sku_id', help='The SKU ID of the Azure plan. Azure plan determines the pricing and service-level agreement of the subscription.  Use 001 for Microsoft Azure Plan and 002 for Microsoft Azure Plan for DevTest.')
-        c.argument('reseller_id', help='Reseller ID, basically MPN Id.')
-
     with self.argument_context('account subscription rename') as c:
         c.argument('subscription_id', help='Subscription Id.')
-        c.argument('subscription_name', help='New subscription name')
+        c.argument('subscription_name', options_list=['--name', '-n', '--subscription-name'], help='New subscription name')
 
     with self.argument_context('account subscription cancel') as c:
         c.argument('subscription_id', help='Subscription Id.')
 
     with self.argument_context('account subscription enable') as c:
         c.argument('subscription_id', help='Subscription Id.')
+
+    with self.argument_context('account subscription list') as c:
+        pass
+
+    with self.argument_context('account subscription show') as c:
+        c.argument('subscription_id', help='The ID of the target subscription.', id_part='subscription')
+
+    with self.argument_context('account subscription list-location') as c:
+        c.argument('subscription_id', help='The ID of the target subscription.')
+
+    with self.argument_context('account tenant list') as c:
+        pass
+
+    with self.argument_context('account alias list') as c:
+        pass
+
+    with self.argument_context('account alias show') as c:
+        c.argument('alias_name', options_list=['--name', '-n'], help='Alias Name')
+
+    with self.argument_context('account alias create', validator=alias_validator) as c:
+        c.argument('alias_name', options_list=['--name', '-n', '--alias-name'], type=str, help='Alias Name')
+        c.argument('display_name', type=str, help='The friendly name of the subscription.')
+        c.argument('workload', arg_type=get_enum_type(['Production', 'DevTest']), help='The workload type of the '
+                   'subscription. It can be either Production or DevTest.')
+        c.argument('billing_scope', type=str, help='Determines whether subscription is fieldLed, partnerLed or '
```

Comment:
why the case isn't consistent? from Swagger directly?

Reply:
It's from swagger. I have updated it.

Reply:
Thanks Feng. better to feedback to swagger, really confusing. :)